### ISSUE TITLE
Describe time period scenes in Observation Window bulleted list

### ISSUE THREAD

**Issue by arouinfar** [phetsims/greenhouse-effect#151]
For #130

Currently, the 2nd bullet point under the Observation Window states the greenhouse gas levels. This is quite awkward when in time period mode, “Ice age levels of greenhouse gases in atmosphere.”

In a recent design meeting we decided to describe the time period using the same description found in the "Currently" statement of the Scene Summary. Please change the 2nd bullet to:
- [ ] The time period is the {{time period}} and there {{scene description}}. 

For example:
* The time period is the {{year twenty twenty}} and there {{are a large number of homes and factories}}. 
* The time period is the {{ice age}} and there {{is a large glacier}}.

**Comment by jbphet** [comment]
I've implemented this.  In the process, I needed to decide between using the existing string and changing the capitalization and adding a new string.  I asked @jessegreenberg, and he said changing the capitalization of a string is okay for now and referenced the following commit:

https://github.com/phetsims/greenhouse-effect/blob/7eaec64e5ba6d59e5eb4c683f8ea2e5cf789b969/js/common/view/describers/TemperatureDescriber.ts#L157-L162

**Comment by jbphet** [comment]
@arouinfar - I've implemented this, assuming I understood it correctly.  Please check it out on master.

Should we consider using the "historically high" terminology here as we've done in other places?  For example, should way say things like, "There are historically low levels of greenhouse gasses in the atmosphere."?

**Comment by arouinfar** [comment]
> Should we consider using the "historically high" terminology here as we've done in other places? For example, should way say things like, "There are historically low levels of greenhouse gasses in the atmosphere."

Oh, that's such a great question @jbphet. We follow a similar format in both the "Currently" statement in the Scene Summary and the first few bullets under the Observation Window. For example:

Scene Summary:
> Currently, the time period is the year twenty twenty and there are a large number of homes and factories. Earth's surface temperature is historically high, 288.1 Kelvin. The sky is cloudy.

Observation Window: 
> * The sunlight is on.
> * The time period is the year twenty twenty and there are a large number of homes and factories.
> * The sky is cloudy.

However, we made the decision to describe what the scene looks like instead of the greenhouse gas levels before we landed on the "historically" language. I think we should describe the greenhouse gas levels for the time periods somewhere, and the logical place would be in the Observation Window. We describe the scene artwork in the Scene Summary and in the responsive descriptions, so I don't think it's necessary to repeat again in the Observation Window.

Sorry to do this to you @jbphet, but let's change that 2nd bullet back to match the slider mode, but the {{value}} would be "Historically high", etc.
* {{Value}} levels of greenhouse gases in atmosphere.
